Add docs for re-enabling FIPS after Strimzi 0.33 upgrade (#7880)
strimzi/strimzi-kafka-operator: documentation/modules/upgrading/proc-switching-to-FIPS-mode-when-upgrading-Strimzi.adoc

  To recreate the certificates with updated algorithms, renew the Cluster and Clients CA.

.. To renew the CAs generated by the Cluster Operator, link:{BookURLUsing}#proc-renewing-ca-certs-manually-str[add the `force-renew` annotation to the CA secrets to trigger a renewal^].
	
.. To renew your own CAs, link:{BookURLUsing}#renewing-your-own-ca-certificates-str[add the new certificate to the CA secret and update the `ca-cert-generation` annotation with a higher incremental value to capture the update^].

. If you use SCRAM-SHA-512 authentication, check the password length of your users.
  If they are less than 32 characters long, generate a new password in one of the following ways:
  
.. Delete the user secret so that the User Operator generates a new one with a new password of sufficient length.

.. If you provided your password using the `.spec.authentication.password` properties of the `KafkaUser` custom resource, update the password in the Kubernetes secret referenced in the same password configuration.
   Don't forget to update your clients to use the new passwords.

> . When all the certificates are using the correct algorithms and the SCRAM-SHA-512 passwords are sufficiently long, you can enable the FIPS mode.

Reviewer: ```suggestion
. Ensure that the CA certificates are using the correct algorithms and the SCRAM-SHA-512 passwords are of sufficient length. You can then enable the FIPS mode.
```

  
. Remove the `FIPS_MODE` environment variable from the Cluster Operator deployment.
  This restarts the Cluster Operator and rolls all the operands to enable the FIPS mode.

> 	After the restart is complete, all Kafka clusters now run with FIPS mode enabled.

Reviewer: ```suggestion
After the restart is complete, all Kafka clusters now run with FIPS mode enabled.
```
